code/obj/machinery/singularity.dm

```diff
@@ -381,7 +381,7 @@ for some reason I brought it back and tried to clean it up a bit and I regret ev
 			if (H.wear_suit)
 				return
 		M.take_toxin_damage(12)
-		M.changeStatus("radiation", 4*(radius+1) SECONDS)
+		M.take_radiation_dose(0.2 SIEVERTS*(radius+1))
```

Comment:
this should... probably be clamped, just in case someone fixes this proc to actually work